opensfm/commands/export_openmvs.py

```diff
@@ -49,6 +49,11 @@ def export(self, reconstruction, graph, data):
 
         for point in reconstruction.points.values():
             shots = graph[point.id].keys()
```

Comment:
Thanks for catching this, @mribbons.
What do you think about this version, which would work for both python 2 and 3
```suggestion
            shots = list(graph[point.id])
```